app/views/tourist_areas/_form.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_for(tourist_area) do |f| %>
+      <%= render 'shared/error_messages', object: f.object %>
+      <%= f.label t".name" %>
+      <%= f.text_field :name, class: 'form-control' %>
```

Comment:
toàn bộ code thống nhất là dùng nháy kép " , thay thế toàn bộ các nháy đơn ' hiện tại thành nháy kép "